### ISSUE TITLE
Takes 15-20 secs to spawn the process

### ISSUE THREAD

**Issue by chinmayembedded** [ildoonet/tf-pose-estimation#417]
Tensor("add_1:0", shape=(?, 6), dtype=float32)
[2019-03-12 16:29:37,384] [TfPoseEstimator-WebCam] [DEBUG] initialization mobilenet_thin : /home/nvidia/chinmay/test_ignore/Pose-estimation/models/graph/mobilenet_thin/graph_opt.pb
[2019-03-12 16:29:37,384] [TfPoseEstimator] [INFO] loading graph from /home/nvidia/chinmay/test_ignore/Pose-estimation/models/graph/mobilenet_thin/graph_opt.pb(default size=432x368)
2019-03-12 16:29:38.220795: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2019-03-12 16:29:38.221024: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-03-12 16:29:38.221128: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0 
2019-03-12 16:29:38.221198: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N 
2019-03-12 16:29:38.221403: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6812 MB memory) -> physical GPU (device: 0, name: Xavier, pci bus id: 0000:00:00.0, compute capability: 7.2)

To spawn the process of run_webcam.py, it takes around 15-20 secs to load the model and allocate the memory. Is there any way to optimize this loading time?

**Comment by Sindreedelange** [comment]
@chinmayembedded, did you figure this out?

**Comment by chinmayembedded** [comment]
@Sindreedelange No, I couldn't figure it out why it the time is spent in executing the code. I implemented openpose(https://github.com/CMU-Perceptual-Computing-Lab/openpose) by the use of python API.

**Comment by Sindreedelange** [comment]
@chinmayembedded, I am currently experimenting with both - did you manage to avoid the startup delay for Openpose? I am currently at about 120 seconds run time for inference on a 5 seconds video (hence my interest in tf-pose).

**Comment by chinmayembedded** [comment]
@Sindreedelange Did you try using cafe version of openpose?

**Comment by Sindreedelange** [comment]
@chinmayembedded, yup, used Caffe version (which I thought was the only one?).  Hmm, so because it tries to allocate too much RAM, initially, it becomes slow?

**Comment by chinmayembedded** [comment]
@Sindreedelange Yes. pretty much the same.

It would be great if you could find the solution. I would be happy helping you
